atd-vzd/schema/atd_txdot_crashes.sql

```diff
@@ -219,84 +216,209 @@ CREATE TABLE public.atd_txdot_crashes (
--- Name: atd_txdot_crashes_geocoded_index; Type: INDEX; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_crashes_geocoded_index; Type: INDEX; Schema: public; Owner: visionzero
 --
 
 CREATE INDEX atd_txdot_crashes_geocoded_index ON public.atd_txdot_crashes USING btree (geocoded);
 
 
 --
--- TOC entry 5631 (class 1259 OID 2346770)
--- Name: atd_txdot_crashes_is_retired_index; Type: INDEX; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_crashes_investigat_agency_id_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_crashes_investigat_agency_id_index ON public.atd_txdot_crashes USING btree (investigat_agency_id);
+
+
+--
+-- Name: atd_txdot_crashes_is_retired_index; Type: INDEX; Schema: public; Owner: visionzero
 --
 
 CREATE INDEX atd_txdot_crashes_is_retired_index ON public.atd_txdot_crashes USING btree (is_retired);
 
 
 --
--- TOC entry 5632 (class 1259 OID 2346771)
--- Name: atd_txdot_crashes_qa_status_index; Type: INDEX; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_crashes_original_city_id_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_crashes_original_city_id_index ON public.atd_txdot_crashes USING btree (original_city_id);
+
+
+--
+-- Name: atd_txdot_crashes_position_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_crashes_position_index ON public.atd_txdot_crashes USING gist ("position");
+
+
+--
+-- Name: atd_txdot_crashes_qa_status_index; Type: INDEX; Schema: public; Owner: visionzero
 --
 
 CREATE INDEX atd_txdot_crashes_qa_status_index ON public.atd_txdot_crashes USING btree (qa_status);
 
 
 --
--- TOC entry 5633 (class 1259 OID 2346795)
--- Name: idx_atd_txdot_crashes_crash_id; Type: INDEX; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_crashes_sus_serious_injry_cnt_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_crashes_sus_serious_injry_cnt_index ON public.atd_txdot_crashes USING btree (sus_serious_injry_cnt);
+
+
+--
+-- Name: atd_txdot_crashes_temp_record_index; Type: INDEX; Schema: public; Owner: visionzero
 --
 
-CREATE UNIQUE INDEX idx_atd_txdot_crashes_crash_id ON public.atd_txdot_crashes USING btree (crash_id);
+CREATE INDEX atd_txdot_crashes_temp_record_index ON public.atd_txdot_crashes USING btree (temp_record);
 
 
 --
--- TOC entry 5634 (class 2620 OID 2346813)
--- Name: atd_txdot_crashes atd_txdot_crashes_audit_log; Type: TRIGGER; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_crashes atd_txdot_crashes_audit_log; Type: TRIGGER; Schema: public; Owner: visionzero
 --
 
-CREATE TRIGGER atd_txdot_crashes_audit_log BEFORE INSERT OR UPDATE ON public.atd_txdot_crashes FOR EACH ROW EXECUTE PROCEDURE public.atd_txdot_crashes_updates_audit_log();
+CREATE TRIGGER atd_txdot_crashes_audit_log BEFORE INSERT OR UPDATE ON public.atd_txdot_crashes FOR EACH ROW EXECUTE FUNCTION public.atd_txdot_crashes_updates_audit_log();
```

Comment:
TIL the difference here. "Procedures do not return a function value; hence CREATE PROCEDURE lacks a RETURNS clause." Very cool.